integreat_cms/api/urls.py

```diff
@@ -44,6 +44,11 @@
         sent_push_notifications,
         name="sent_push_notifications",
     ),
+    path(
+        "fcm/",
```

Comment:
Since the existing endpoint `sent_push_notifications` isn't documented anywhere, we could also just replace the existing one instead of adding a new one...

Reply:
Agreed.